### ISSUE TITLE
Documentation update suggestion for README.md in 5-app-infra

### ISSUE THREAD

**Issue by marktomlinson1** [terraform-google-modules/terraform-example-foundation#614]
### TL;DR

Instructions for Steps 7 and 8 under "Run Terraform locally" section of 5-app-infra README.d would benefit from further clarity. 

### Expected behavior

README provides unambigious instructions to configure Service Account Token Creator role. 

### Observed behavior

Current documentation in README.md for local execution is difficult to follow.

Step 7 would benefit from a review for clarity:

> 7. Provide the user that will be running ./tf-wrapper.sh the Service Account Token Creator role to the bu1 project service accounts

Step 8 appears to be a repeat of the previous instruction. Given the number of environments and projects now created, it would also benefit from a clearer explanation of which PROJECT_SERVICE_ACCOUNT and PROJECT to use, or if it needs to be repeated for each environment.  

> 8. Provide the user permissions to run the terraform locally with the serviceAccountTokenCreator permission.
gcloud iam service-accounts add-iam-policy-binding $PROJECT_SERVICE_ACCOUNT --project $PROJECT --member="user:$(gcloud auth list --format="value(account)")" --role="roles/iam.serviceAccountTokenCreator"

### Terraform Configuration

```hcl
As per 5-app-infra
```

### Terraform Version

```sh
Terraform v0.13.0
```

### Additional information

As I was running locally in Cloud Shell under admin@ with Owner/Org Admin roles, not sure if these changes were needed.
